readme.md

```diff
@@ -12,7 +12,9 @@ Copy both files in `apis/` to `/apis/`, and put `programs/github` wherever you w
 Usage
 -----
 
-    github clone <user>/<repo name> [destination folder]
+    github clone <user>/<repo name> [branch|tag] [destination folder]
+
+The branch or tag argument is optional, and defaults to 'master'.
```

Comment:
I said "commit sha", not object sha. I'm also confused by tree shas vs commit shas. My understanding was that a commit object contained a single tree, and so tree and commit shas are different things?